src/common/entryPoint.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import {TelemetryHelper} from "../common/telemetryHelper";
+import {Telemetry} from "../common/telemetry";
+import {Log} from "../common/Log";
+import {OutputChannel} from "vscode";
+
+/* This class should we used for each entry point of the code, so we handle telemetry and error reporting properly */
+export class EntryPoint {
```

Comment:
The name here is a little confusing. It should reflect the purpose of the class, which is telemetry and error reporting.